hack/autobuild.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+# Copyright © 2021 Alibaba Group Holding Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# -----------------------------------------------------------------------------
+#The script format is similar to that of auto build：
+#If we need to make kubernetes:1.20.14 image，We just need to enter: kubernetes v1.20.14 amd64
+
+echo "please enter image name:"
+read  imageName   #eg: kubernetes v1.20.14 amd64
```

Comment:
Read from stdin is not good. Just using `autobuild.sh kubernetes v1.20.14 amd64`  then using $1 $2 $3 to get args in your script